backend/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# this Dockerfile won't work on it's own, as it requires there to be a
+# mongodb database running, and a corresponding configuration.
+# use the docker compose file in the root directory instead.
+# if for some reason you want to use this Dockerfile/image on its own,
+# uncomment the MONGO_URL line below and insert the relevant information
+# (remember that you'll also need to make the mongodb database you use
+# available to the Docker containers, possibly by port bindings). 
+FROM python:3.10-slim-buster
+
+WORKDIR /app
+
+COPY requirements.pip requirements.pip
+
+RUN pip3 install -r requirements.pip
+
+COPY . .
+
+# ENV FLASK_BIND_IP=0.0.0.0
```

Comment:
It makes sense to include the .env file in the repo then. I will add this in the master branch